account/account.go

```diff
@@ -103,6 +103,16 @@ func GetAccount(ctx context.Context, cfg *config.Configuration, fetcher stored_r
 		return nil, errs
 	}
 
+	errs = account.Privacy.IPv6Config.Validate(errs)
+	if len(errs) > 0 {
+		return nil, errs
+	}
```

Comment:
This check may be controversial. The preferred approach we now take to account config error handling is to record a metric and revert to defaults such that auctions can still run using as much of the account config that's valid. The validation may be better suited for the `/ accountID resolved to a valid account, merge with AccountDefaults for a complete config` branch.

I assume we don't need to validate the default account because that's handled separatly on app start?

Reply:
Is it ok if metrics will be added in a separate PR? I modified account validation and in case data is not valid we set it to default.

Default config can be overwritten by host config. I assume we still need a host config ipV4 and ipV6 validation. I'm ok to remove host config validation

Reply:
Absolutely.